TIL/topics/dart_basics/ModelClass.md

```
@@ -0,0 +1,58 @@
+				final int age;
+				const Person({this.naem, this.age});
+			}
+		```
+- 값 변경을 위해서 copyWith()를 제공
+	- 불변 객체의 필드 값을 수정한 객체 복사를 위해 copyWith 메서드를 만든다.
+		```dart
+		class Person {
+			final String name;
+			final int age;
+			...
+			Person copyWith({String? name, int? age}) {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)